code/game/machinery/cell_charger.dm

```diff
@@ -91,7 +91,7 @@
 		if(!charging || (stat & (BROKEN|NOPOWER)) || !anchored)
 			return
 
-		use_power(2000)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
-		charging.give(1750)	//inefficiency.
+		use_power(200)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
+		charging.give(175)	//inefficiency.

```

Comment:
> it's why you see things like SMESes increasing total available power to the grid just by existing.

That's actually working correctly. If you have an alternator putting out 50 amps peak, and have a battery with a 300 amp peak, you will have 350 amps available total on the bus (if it's a 12v system, that's 4,200 watts). That is, until the battery discharges, then you only have 50 amps available. The same with SMES: if you hotwire them, you have the output of both the SMES AND the engine available... until the SMES are discharged...

I can tell you though, until something like voltage is implemented, some of the other weird oddities that come up won't go away.